packages-proprietary/tokens/src/components/ams/file-input.tokens.json

```diff
@@ -10,7 +10,7 @@
       "font-family": { "value": "{ams.typography.font-family}" },
       "font-size": { "value": "{ams.typography.body-text.font-size}" },
       "font-weight": { "value": "{ams.typography.body-text.font-weight}" },
-      "line-height": { "value": "{ams.typography.body-text.line-height}" },
+      "line-height": { "value": "{ams.inputs.line-height}" },
```

Comment:
Shouldn't this be on `file-selector-button` as well? The button is too small here I think

Reply:
Yes, good find, the difference was less than 2px 😅
The line hight on the file input itself doesn’t even do much.

Reply:
No, the button is always taller than a single line of text. But it's safer to keep it, in case the text wraps.